homeassistant/components/squeezebox/strings.json

```
@@ -103,5 +103,16 @@
         "unit_of_measurement": "[%key:component::squeezebox::entity::sensor::player_count::unit_of_measurement%]"
       }
     }
+  },
+  "options": {
+    "step": {
+      "init": {
+        "title": "LMS Configuration.",
+        "data": {
+          "browse_limit": "Number of items to return from media browser.",
+          "volume_step": "Step when increasing or decreasing volume."
```

This is more consistent with how we describe the services (`volume_up` and `volume_down`). "Step" is not a concept defined in the HA docs, so let's use more descriptive language:

```suggestion
          "volume_step": "Amount to adjust the volume when turning volume up or down."
```

---

Thanks